sources/install/package_base.sh

```diff
@@ -243,6 +270,37 @@ function install_gf() {
     add-to-list "gf,https://github.com/tomnomnom/gf,A wrapper around grep to avoid typing common patterns"
 }
 
+function install_java11() {
+    # CODE-CHECK-WHITELIST=add-history,add-aliases
+    colorecho "Installing java 11"
+    mkdir -p /tmp/java
+    cp -r /root/source/assets/java/java11 /tmp/java
+    cd /tmp/java/java11
+    dpkg -i openjdk-11-jre-headless_11.0.20+8-1\~deb11u1_amd64.deb
+    dpkg -i openjdk-11-jdk-headless_11.0.20+8-1\~deb11u1_amd64.deb
+    dpkg -i openjdk-11-jre_11.0.20+8-1\~deb11u1_amd64.deb
+    dpkg -i openjdk-11-jdk_11.0.20+8-1\~deb11u1_amd64.deb
```

Comment:
Why are the files static and not dynamically downloaded?

Reply:
Do debian repositories have a lifespan?